core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataIndexAliasesService.java

```diff
@@ -88,12 +88,13 @@ protected ClusterStateUpdateResponse newResponse(boolean acknowledged) {
 
                 @Override
                 public ClusterState execute(ClusterState currentState) {
-                    return innerExecute(currentState, request.actions());
+                    return updateAliasesInClusterState(currentState, request.actions());
                 }
             });
     }
 
-    ClusterState innerExecute(ClusterState currentState, Iterable<AliasAction> actions) {
+    // public for use in tests
+    public ClusterState updateAliasesInClusterState(ClusterState currentState, Iterable<AliasAction> actions) {
```

Comment:
Does it just need to be visible, or overridable? If not the latter, can you make it final?